docs/source/general/setup.rst

```diff
@@ -179,12 +188,24 @@ Run tests with:
 
     ./runtests.py
 
+Troubleshooting
+---------------
+
+If you encounter any issue during installation, run:
+
+.. code-block:: shell
+
+    pip install -r requirements.txt -r requirements-test.txt instead of pip install -r requirements-test.txt
```

Comment:
Let's make this copy-paste friendly.

Reply:
sorry I did it probably by coincidence however do you have any idea what @atb00ker meant: https://github.com/openwisp/django-freeradius/issues/253#issuecomment-577761851

Reply:
you should remove `instead of pip install -r requirements-test.txt`